### DISCUSSION TITLE
citus_rebalance_start() ERROR

### DISCUSSION THREAD

**Discussion by diogoaraujodtx** [citusdata/citus#7584] [Q&A] [answered]
Hi guys!

I am using a citus cluster with 2 workers. At this point, I want to add a new worker and redistribute the tables among the 3 workers. I used citus_rebalance_start() function but it returns an error (ERROR:  connection to the remote node localhost:5432 failed with the following error: fe_sendauth: no password supplied). I have already tried to change pg_hba.conf file and still doesn't work. Can someone help me?

Thanks

**Comment by JelteF** [comment]
what does your pg_hba.conf look like now?

**Reply by diogoaraujodtx** [reply]
Already did it correctly and solve it. Meanwhile, a question arose about Citus. I have a cluster with 1 coordinator and 2 workers and created a distributed table 'student' (with **create_distributed_table(student, id)**) with 100 records. This function will divide the table in shards and distribute among the workers. But when I access a worker and make "select count(*) from student" it returns 100 too. I was expecting another value <100 because it was supposed to have only a part of the initial table. Can you understand why?
ty!

**Reply by JelteF** [reply]
The `student` table on the worker is not one of the shards. If you want to see the shards you can set:
```sql
SET citus.show_shards_for_app_name_prefixes = '*';
```

And if you then look at the existing tables you can see the underlying shard tables. These are named something like `student_123456`

**Reply by JelteF** [reply]
Also could you share how you solved your problem?

**Reply by diogoaraujodtx** [reply]
right, but if table `student` is distributed, how can count of rows in a worker return the initial table size (100)? Should't be a value less than 100 resulting by the sum of rows of tables student_shardid in that worker?

**Reply by diogoaraujodtx** [reply, marked as answer]
sure, I solve my problem changing line `host all all 127.0.0.1/32 scram-sha-256` to `host all all 127.0.0.1/32 trust` in the `pg_ha.conf` file

**Reply by JelteF** [reply]
The result is 100 on all workers, because when you query `student` you don't query the shards directly and the worker will act like a coordinator in the sense that it will send queries to all workers. If you query `student_123456` you will get the number of entries in that shard.

**Reply by JelteF** [reply]
See here for more details: https://www.citusdata.com/blog/2022/06/17/citus-11-goes-fully-open-source/#any-node
